electron/main/llm/ipcHandlers.ts

```diff
@@ -44,13 +44,18 @@ export const registerLLMSessionHandlers = (store: Store<StoreSchema>) => {
         event.sender.send('anthropicTokenStream', chatHistory.id, chunk)
       }
 
+      const transformedChatHistory: ChatCompletionMessageParam[] = chatHistory.displayableChatHistory.map((message) => {
```

Comment:
What was the reason for this change out of interest? 

Something to note is that we are going to move the LLM calling logic all to the renderer process, it'll make it so much easier to not have to deal with all this messy logic with chunks being passed via IPC 😅